docs/examples/resampling.ipynb

```diff
@@ -0,0 +1,219 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example - Resampling\n",
+    "\n",
+    "API Reference for `rio.clip`:\n",
```

Comment:
I see `rio.clip`, but it is for `rio.reproject`. If you copied from another example. I wonder if that is an issue there too?

Reply:
Rah sorry, this is me doing to many things in the same time... 😓 
I copied from clip, don't worry

Reply:
Phew, thanks :sweat_smile: